tools/BaseRunner.py

```diff
@@ -8,6 +10,16 @@ def set_process_limits():
     resource.setrlimit(resource.RLIMIT_DATA, (4 << 30, 4 << 30))
 
 
+def kill_child_processes(parent_pid, sig=signal.SIGTERM):
```

Comment:
How about changing this to a `signal.SIGKILL` here, since `SIGTERM` can be intercepted and ignored by the process? Especially since further down you use `proc.kill()` to kill the parent.

Reply:
Good point

Reply:
That's right.  I'll fix it.